### PR TITLE
Add neovim editor configuration example

### PR DESCRIPTION
**Pull request by chaudum**
Signed-off-by: Christian Haudum <christian.haudum@gmail.com>

### CODE REVIEW

File: editor/neovim/init.lua [hunk 1]
```diff
@@ -0,0 +1,5 @@
+local lspconfig = require 'lspconfig'
+
+lspconfig.jsonnetls.setup{
+  cmd = { '~/go/bin/jsonnet-language-server' };
```
Comment (jdbaldry) [inline editor/neovim/init.lua:4]: nit: Perhaps this should be based on GOPATH to be more general?

**Review by jdbaldry — APPROVED** [review]
Looks reasonable to me but I know very little about Neovim configuration. Thanks for contributing this!

Do we need to set `JSONNET_PATH` anywhere for this to work?

**Comment by chaudum** [comment]
> Looks reasonable to me but I know very little about Neovim configuration. Thanks for contributing this!
> 
> Do we need to set `JSONNET_PATH` anywhere for this to work?

I guess if you want to use libraries outside of the `root_dir` (https://github.com/jdbaldry/jsonnet-language-server/pull/26/files#diff-0f0d66d645b4ed68aabd337810566493f2fa91ea55d4cca223ae80a0d9094e91R8-R10) you need to export your JSONNET_PATH so the language server binary can pick it up.

**Comment by chaudum** [comment]
Can be closed in favour of using the official [lspconfig](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/jsonnet_ls.lua).

**Comment by chaudum** [comment]
> Looks reasonable to me but I know very little about Neovim configuration. Thanks for contributing this!
> 
> Do we need to set `JSONNET_PATH` anywhere for this to work?

There is a PR to the official neovim/nvim-lspconfig repo which adds JSONNET_PATH to the language server config: https://github.com/neovim/nvim-lspconfig/pull/1437

**Pull request closed without merging**